avro-fastserde/src/jmh/java/com/linkedin/avro/fastserde/FastAvroSerdesBenchmark.java

```diff
@@ -81,30 +100,48 @@ public void prepare() throws Exception {
     generatedRecord = (GenericData.Record) generator.generate(properties);
     serializedBytes = serializeGeneratedRecord(generatedRecord);
 
-    serializer = new AvroGenericSerializer(benchmarkSchame);
-    fastSerializer = new AvroGenericSerializer(new FastGenericDatumWriter<>(benchmarkSchame));
-    deserializer = new AvroGenericDeserializer<>(new GenericDatumReader<>(benchmarkSchame));
-    fastDeserializer = new AvroGenericDeserializer<>(new FastGenericDatumReader<>(benchmarkSchame));
+    serializer = new AvroGenericSerializer(benchmarkSchema);
+    fastSerializer = new AvroGenericSerializer(new FastGenericDatumWriter<>(benchmarkSchema));
+    deserializer = new GenericDatumReader<>(benchmarkSchema);
+    fastDeserializer = new FastGenericDatumReader<>(benchmarkSchema);
   }
 
   @Benchmark
-  public void testAvroSerialization() throws Exception {
-    // use vanilla avro 1.4 encoder, do not use buffer binary encoder
-    serializer.serialize(generatedRecord);
+  @OperationsPerInvocation(NUMBER_OF_OPERATIONS)
+  public void testAvroSerialization(Blackhole bh) throws Exception {
+    for (int i = 0; i < NUMBER_OF_OPERATIONS; i++) {
+      // use vanilla avro 1.4 encoder, do not use buffer binary encoder
+      bh.consume(serializer.serialize(generatedRecord));
```

Comment:
I think the `AvroGenericSerializer ` util function is also using binary buffer by default?